modules/registry-operator-config-resources.adoc

```diff
@@ -45,6 +45,16 @@ data:
 <1>  If the registry has the port, such as `registry-with-port.example.com:5000`.
 : should be replaced with `..`.
 
+. To configure an additional CA:
++
+----
+$ oc create configmap registry-config --from-file=<cert> -n openshift-config
```

Comment:
Should add external registry address after --from-file, like
$oc create configmap registry-config --from-file=<external_registry_address>=ca.crt -n openshift-config